package.json

```diff
@@ -32,12 +32,14 @@
   "main": "./src/main/main.js",
   "scripts": {
     "dev": "gulp && electron-webpack dev",
-    "postinstall": "electron-builder install-app-deps buildDependenciesFromSource",
+    "install-app-deps": "electron-builder install-app-deps buildDependenciesFromSource",
+	"install-all": "yarn && yarn install-app-deps",
```

Comment:
@sk91 curious, why did you remove `postinstall`?  Cheers

Reply:
@rodrigopavezi it takes a lot of time becouse it rebuilds everything from source. For unit tests it is not needed

Reply:
@sk91 makes sense. cheers